tools/blobtoolkit/blobtoolkit.xml

```diff
@@ -156,7 +148,7 @@
                 #end if
                 ./Blobdir
             #end if
-            && tar -C ./Blobdir -zcvf Blobdir.tgz .
+            && tar -zcf './Blobdir.tgz' './Blobdir'
```

Comment:
why is here blobdir?

What are you trying here? A folder in the tarball or no folder in the tarball?

Reply:
Yes, it includes the files inside a folder. In that way the interactive tool seems to work correctly

Reply:
Of course, we need to adopt the interactive tool to NOT take the folder.

The question is what is more user-friendly. What is easier to explain to users and then we adopt both tools.

Reply:
Yes, I agree, but from the users point of view I don't think there's a great difference if compressing the files with without the folder.

Reply:
How should the user know that a folder is needed?

Reply:
Well, this dataset is generated by the tool itself. If they want to upload a blobdir folder just to use the interactive tool, I think it would be possible to include that information in the help section perhaps?

Reply:
That is the big question here. And yes I think that is a use-case. People with data, that just want to visualise them.

Reply:
Ok, I'm modifying the interactive tool in order to avoid those changes, probably it will be more clear in that way. Also I'll include some info there.

Reply:
Also it should work https://github.com/usegalaxy-eu/galaxy/pull/158